oauth-function-template/datastore/queries.js

```diff
@@ -81,10 +81,20 @@ let REFRESH_TOKEN_COLUMN = {
   }
 };
 
+let ERROR_CODE_COLUMN = {
+  input: {
+    columnType: "TEXT",
+    name: "errorCode",
```

Comment:
1. As per our today's discussion - do we really want this column to be named `ERROR_CODE_COLUMN`?
2. error code is going to be text message or number converted to string?

Reply:
text message

Reply:
number